RELEASE_NOTES.md

```diff
@@ -1,5 +1,68 @@
+- Fixed a bug where detaching from a container using the detach keys would cause Podman to hang until the container exited
+- Fixed a bug where `podman create --rm` did not work with `podman start --attach`
+- Fixed a bug where invalid named volumes specified in `podman create` and `podman run` could cause segfaults ([#2301](https://github.com/containers/libpod/issues/2301))
+- Fixed a bug where the `runtime` field in `libpod.conf` was being ignored. `runtime` is legacy and deprecated, but will continue to be respected for the forseeable future
+- Fixed a bug where `podman login` would sometimes report it logged in successfully when it did not
+- Fixed a bug where `podman pod create` would not error on receiving unused CLI argument
+- Fixed a bug where rootless `podman run` with the `--pod` argument would fail if the pod was stopped
+- Fixed a bug where `podman images` did not print a trailing newline when not invoked on a TTY ([#2388](https://github.com/containers/libpod/issues/2388))
+- Fixed a bug where the `--runtime` option was sometimes not overriding `libpod.conf`
+- Fixed a bug where `podman pull` and `podman runlabel` would sometimes exit with 0 when they should have exited with an error ([#2405](https://github.com/containers/libpod/issues/2405))
+- Fixed a bug where rootless `podman export -o` would fail ([#2381](https://github.com/containers/libpod/issues/2381))
+
+### Misc
+- Podman migrated to a new, shared memory locking model in this release. As part of this, if you are running Podman with pods or dependency containers (e.g. `--net=container:`), you should run the `podman system renumber` command to migrate your containers to the new model
+- Podman migrated to a new command-line parsing library, and the output format of help and usage text has somewhat changed as a result
+- Updated vendored Buildah to v1.7, picking up a number of bugfixes
+- Updated vendored containers/image library to v1.4, picking up a number of bugfixes and performance improvements to pushing images
+- Updated vendored containers/storage library to v1.10, picking up a number of bugfixes
+- Work on the remote Podman client for interacting with Podman remotely over Varlink is progressing steadily, and many image and pod commands are supported
+- Added path masking to mounts with the `:z` and `:Z` options, preventing users from accidentally performing an SELinux relabel of their entire home directory
+- The `podman container runlabel` command will not pull an image if it does not contain the requested label
+- Many commands' usage information now includes examples
+- `podman rm` can now delete containers in containers/storage, which can be used to resolve some situations where Podman fails to remove a container
+- The `podman search` command now searches multiple registries in parallel for improved performance
+- The `podman build` command now defaults `--pull-always` to true
+
 ## 1.0.0
```

Comment:
I think you should mention the conversion to Cobra in here somewhere.  I'd suggest Features, but at least in Misc.

Reply:
There's a note on it in misc